common/men_at_arms_types/00_regional_maa_types.txt

```diff
@@ -1 +1,70 @@
-﻿# Warcraft
\ No newline at end of file
+﻿# standard costs
+@maa_buy_cost = 150
+@maa_low_maintenance_cost = 1.0
+@maa_high_maintenance_cost = 5.0
+#Warcraft
+war_camel_rider = {
+	type = light_cavalry
+	
+	damage = 22
+	toughness = 15
+	pursuit = 20
+	screen = 20
+	
+	terrain_bonus = {
+		desert = { damage = 20 toughness = 8 }
+		drylands = { damage = 15 toughness = 8 }
+		oasis = { damage = 15 }
+		floodplains = { damage = 5 }
+		hills = { damage = -10 }
+		desert_mountains = { damage = -10 pursuit = -10 }
+		mountains = { damage = -15 pursuit = -20 }
+		wetlands = { damage = -15 toughness = -10 pursuit = -20 screen = -20 }
+	}
+	
+	counters = {
+		archers = 1
+		caster = 1
+	}
+	
+	buy_cost = { gold = camel_rider_recruitment_cost }
+	low_maintenance_cost = { gold = camel_rider_low_maint_cost }
+	high_maintenance_cost = { gold = camel_rider_high_maint_cost }
+	
+	stack = 100
+	ai_quality = { value = culture_ai_weight_camels }
+	icon = war_camels
+	fallback_in_hired_troops_if_unlocked = yes
+}
+
+war_elekk_rider = {
```

Comment:
Keep the vanilla name, it's easier to handle.

Reply:
elephant_rider?

Reply:
> elephant_rider

It's called `war_elephant` in the vanilla.